metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/MediaProcessPluginMetadata.java

```diff
@@ -1,12 +1,16 @@
 package eu.europeana.metis.core.workflow.plugins;
 
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * @author Simon Tzanakis (Simon.Tzanakis@europeana.eu)
  * @since 2018-04-20
  */
 public class MediaProcessPluginMetadata extends AbstractMetisPluginMetadata {
```

Comment:
This class is now exactly the same as LinkCheckingPluginMetadata. Maybe make it one shared class for the two plugins?